docs/hardware.md

```diff
@@ -473,11 +473,21 @@ VirtualBox is a general-purpose full virtualiser for x86 hardware, targeted at s
 
 The VirtualBox virtual machine is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 PC/server and still get the same great DietPi features and experience.
 
-### Parallels Desktop
+### Parallels
 
-![macOS Parallels Desktop](assets/images/Parallels3.png){: width="270" height="82" loading="lazy"}
+![Parallels Desktop](assets/images/Parallels3.png){: width="270" height="82" loading="lazy"}
 
-The Parallels Desktop virtual machine for macOS is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 macOS system (e.g. Mac mini 2011/2012/2014/2018) and still get the same great DietPi features and experience.
+The [Parallels Desktop virtual machine](https://www.parallels.com/products/desktop/) is great for those occasions where SBC performance just isn't enough. Run one of these e.g. on any x86_64 macOS system (e.g. Mac mini 2011/2012/2014/2018) and still get the same great DietPi features and experience.
+
+### UTM
+
+![UTM](assets/images/UTM3.png){: width="270" height="221" loading="lazy"}
+
+The [UTM virtual machine](https://mac.getutm.app/) is great for those occasions where SBC performance just isn't enough. Run one of these on any macOS system (e.g. Mac mini 2011/2012/2014/2018 and M1 Apple Silicon) and still get the same great DietPi features and experience.
+
+!!! info "Emulation feature of UTM"
+    The emulation feature is an option of the virtualization technique to run x86 code on a M1 Mac system and vice versa a M1 Mac code on a x86 Mac system.  
```

Comment:
```suggestion
    The emulation feature is an option of the virtualization technique to run x86 images on an M1 Mac system and vice versa a M1 Mac images on an x86 Mac system.  
```

Reply:
Because code is not architecture dependent, but the compiled binaries are, resp. the OS images. And I learned not too long ago that you use "an" instead of "a" when the next word is spoken like it begins with a vocal, not only when the first letter really is vocal 🙂.